woocommerce/woocommerce
# [Changefile] Fix Message regex

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/48629

Updates the changefile message regex to only include the message.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Since the files touched here don't produce a changelog, the best way to test is locally. Make the following change to log out the PR details but skip the rest, which involves cloning the repo and trying to produce a changefile, if required.
```diff
diff --git a/tools/monorepo-utils/src/changefile/index.ts b/tools/monorepo-utils/src/changefile/index.ts
index f9dc829d84..886f49534c 100644
--- a/tools/monorepo-utils/src/changefile/index.ts
+++ b/tools/monorepo-utils/src/changefile/index.ts
@@ -77,6 +77,8 @@ const program = new Command( 'changefile' )
                        }
 
                        const details = getChangelogDetails( prBody );
+                       console.log( details );
+                       return;
                        const { significance, type, message, comment } = details;
                        const changelogDetailsError = getChangelogDetailsError( details );
```
2. Build the tool, `pnpm --filter=@woocommerce/monorepo-utils start`
3. Use this PR's description to test. See the message I've added to below to the template.
4. `pnpm utils changefile 48649`. See the logged out message, is it correct?
5. See that tests pass.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

This PR fixes messages in the template!!!!

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
